data/afrique/homePage/index.json

```diff
@@ -480,52 +480,40 @@
         "visualProminence": "NORMAL",
         "radioSchedule": [
           {
-            "serviceId": "bbc_afrique_radio",
-            "publishedTimeStart": "2024-01-22T06:00:00.000Z",
-            "publishedTimeEnd": "2024-01-22T06:15:00.000Z",
-            "publishedTimeDuration": "PT15M",
-            "title": "Le Journal",
-            "brandPid": "p03p35rt",
-            "episodeTitle": "22/01/2024 GMT",
-            "episodePid": "w172z9v5y9h5lcf",
-            "broadcastPid": "p0h06c79",
-            "summary": "Le premier rendez-vous d'information de la matinée."
+            "id": "p086pmtv",
```

Comment:
Can we get an update-to-date version of the afrique homepage here, based on Pedram's branch, so that we can see the radio schedule properly in Storybook?

Ideally one with a live / next & 2 on-demand episodes

Reply:
I have added the fixture data from Pedram's branch, but deleted the two curations which were for the banners that aren't supported yet as they caused a home page tests to fail. I could instead add to the test a check for only the count of valid curations in the data to compare against the number of sections on the page (instead of counting all curations with summaries and comparing to the number of sections), but then we would be mixing in validation code in higher level than where it is tested itself and gets a bit messy.
When this PR gets merged https://github.com/bbc/simorgh/pull/11323/files I could put the curations back in so that it is testing those types of curation too. I think this is the best way to do it.